src/applications/appeals/10182/tests/10182-notice-of-disagreement.cypress.spec.js

```diff
@@ -25,13 +33,55 @@ const testConfig = createTestConfig(
             .click();
         });
       },
+      'eligible-issues': () => {
+        cy.get('@testData').then(data => {
+          data.additionalIssues.forEach((item, index) => {
+            cy.get('.va-growable-add-btn')
+              .first()
+              .click();
+
+            cy.get(`input[name^="root_additionalIssues_${index}"]`)
+              .first()
+              .clear()
+              .type(item.issue);
+            const date = item.decisionDate.replace(/-0/g, '-').split('-');
+            cy.get(`select[name$="${index}_decisionDateMonth"]`).select(
+              date[1],
+            );
+            cy.get(`select[name$="${index}_decisionDateDay"]`).select(date[2]);
+            cy.get(`input[name$="${index}_decisionDateYear"]`)
+              .clear()
+              .type(date[0]);
+            cy.get('.update')
+              .first()
+              .click();
+          });
+          cy.get('input[name="root_socOptIn"]').check();
+        });
+      },
     },
 
     setupPerTest: () => {
-      // Log in if the form requires an authenticated session.
-      // cy.login();
+      cy.login(mockUser);
```

Comment:
Not sure if you need a mockUser with specific data, but you can also call `cy.login()` without any parameters and it will mock a user for you.